eth/catalyst/api.go

```diff
@@ -141,8 +142,10 @@ func (api *consensusAPI) AssembleBlock(params assembleBlockParams) (*executableD
 		Extra:      []byte{},
 		Time:       params.Timestamp,
 	}
-	if config := api.eth.BlockChain().Config(); config.IsLondon(header.Number) {
-		header.BaseFee = misc.CalcBaseFee(config, parent.Header())
+	if config := api.eth.BlockChain().Config(); config.IsBanach(header.Number) {
+		stateDb, _ := api.eth.BlockChain().StateAt(parent.Root())
```

Comment:
I have seen this pattern in multiple places, why are we ignoring the errors?

Reply:
You're right, I think this is an artifact from earlier draft code, will fix.

Reply:
Fixed in [af385f6](https://github.com/scroll-tech/go-ethereum/pull/634/commits/af385f6343df9c968c42ba8037923da8f35c38fc).